application/review/views.py

```diff
@@ -12,8 +12,14 @@
 @review_blueprint.route("/<review_token>")
 def review_page(review_token):
     try:
+        # TODO: Once all guid-based tokens have expired adjust this to only filter by ID & token.
         id, version = decode_review_token(review_token, current_app.config)
-        measure_version = MeasureVersion.query.filter_by(guid=id, version=version, review_token=review_token).one()
+        measure_version = MeasureVersion.query.filter_by(
+            guid=id, version=version, review_token=review_token
+        ).one_or_none()
+        if not measure_version:
```

Comment:
```suggestion
        if not measure_version and version is None:
```

If we leave this as-is, and someone tries to decode a review token generated by the old system after it's been replaced by a review token from the new system, we generate a DataError as we try to select on id with a guid (e.g. 'CXT 003'), which is a string vs the id's integer.

This is super unlikely to ever happen, but it's an easy mitigation to put in place, so 🤷‍♂️

Reply:
I *think* I've mitigated this now, but take a look.